client/extensions/woocommerce/woocommerce-services/api/url.js

```diff
@@ -0,0 +1,4 @@
+const namespace = 'connect/';
```

Comment:
Is this code likely to be edited / maintained at the same time as client/extensions/woocommerce/state/sites/request.js ? If so, let's choose a different variable name than `namespace` to avoid confusion